x/wasm/keeper/querier_test.go

```diff
@@ -1047,3 +1047,60 @@ func TestEnsurePaginationParams(t *testing.T) {
 		})
 	}
 }
+
+func TestQueryBuildAddress(t *testing.T) {
+	specs := map[string]struct {
+		srcQuery *types.QueryBuildAddressRequest
```

Comment:
```suggestion
		src *types.QueryBuildAddressRequest
```
personal preference, just to keep consistent with all the others

Reply:
done here: https://github.com/CosmWasm/wasmd/pull/1753/commits/e405904b253c16e3cfd8191dfda5b63591ba565d